File: test/e2e/navigation_spec.js
'use strict';

describe('Navigation Menu', function() {

  it('changes active link depending on route', function() {
    browser.get('/');
    var activeListItem = element(by.css('.active')).getText().then(function(name) {
      expect(name).toBe('Home');
    });

    var edgesLink = element(by.linkText('Edges'));
    edgesLink.click();
    activeListItem = element(by.css('.active')).getText().then(function(name) {
      expect(name).toBe('Edges');
    });

  });

});

File: spec/features/navigation_spec.rb
require 'rails_helper'

describe "Navigation Menu", js: true do
	it "changes active menu element depending on route" do
		visit '/'
		expect(find('.active').find('a').text).to eq('Home')

		click_link('Edges')
		expect(find('.active').find('a').text).to eq('Edges')
	end
end